docs/index.md

```diff
@@ -26,12 +39,39 @@ In future the provider will be extended to support more of Juju's capabilities.
 
 The provider has a dependency on Juju CLI configuration store.
 
-In order to operate, the provider expects configuration to be found in one of the following locations:
+## Authentication
 
-* `$XDG_DATA_HOME/juju`
-* `~/.local/share/juju`
+There are 3 ways to define credentials for authentication with the juju controller you wish to use. They are displayed in order in which the provider looks for credentials.
 
-The intention is to remove this dependency in future.
+### Static credentials
+
+Define the juju controller credentials in the provider definition in your terraform plan.
```

Comment:
I'd spell "juju" as "Juju"  as we're not just speaking about the CLI.

Reply:
Also, regarding "controller credentials" -- same note as above. I'd say "cloud credentials" or "the cloud credentials you want to use with your Juju controller..."